.github/workflows/build_docker_on_release.yml

```diff
@@ -75,13 +75,12 @@ jobs:
             org.opencontainers.image.revision=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.revision'] }}
             org.opencontainers.image.version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
 
-      # TODO: check if we can exclude this commit from the changelog
```

Comment:
@surchs, are you still interested in doing this?

If so, I think the easiest ways would be to update [.autorc](https://github.com/neurobagel/workflows/blob/d95a01771b2a79a27abed655057e3cacb43bc66f/template_configs/.autorc#L11-L18) to either add `neurobagel-bot[bot]` to the `omit-commits` author list, or use a prefix like `[skip]` in the commit message.